main/src/ca/uwaterloo/flix/language/phase/Monomorpher.scala

```diff
@@ -446,17 +444,14 @@ object Monomorpher {
   }
 
   /**
-    * Performs specialization of the given expression `exp0` under the environment `env0` w.r.t. the
-    * given substitution `subst`.
+    * Specializes `exp0`, renaming variables according to `env0` and specializes
+    * w.r.t. `subst`.
     *
-    * Replaces every reference to a parametric function with a reference to its specialized version.
+    * Replaces every polymorphic function reference with a reference to a specialized version.
     *
     * Replaces every local variable symbol with a fresh local variable symbol.
-    *
-    * If a specialized version of a function does not yet exist, a fresh symbol is created for it,
```

Comment:
Is this no longer true?

Reply:
Its true for many functions and is explained in the main text so i don't think it should be repeated every time